dsgrid/dataset/dataset_schema_handler_base.py

```diff
@@ -278,8 +305,16 @@ def get_time_zone_mapping(self):
         """
 
     @track_timing(timer_stats_collector)
-    def _convert_time_dimension(self, load_data_df, time_zone_mapping):
+    def _convert_time_dimension(self, load_data_df):
```

Comment:
I wonder if we should keep `_convert_time_dimension(self, load_data_df, time_zone_mapping)` and have time_zone_mapping retrieval be a different func instead because, as you pointed out yesterday, the mapping could come from project level geography.

Reply:
This point makes sense to me. Relatedly, is it always clear how to get_time_zone_mapping from the dataset? 

I wonder if there is or should be a project-specific way to do this that is distinct from the dataset-specific way?